qemu.robot

```diff
@@ -5,7 +5,9 @@ Resource  resources/qemu.robot
 Resource  resources/resinos.robot
 Resource  resources/kernel.robot
 Test Timeout  30 minutes
-Suite Teardown    Run Keywords    Terminate All Processes    kill=True
+Suite Teardown    Run Keywords    Get "journalctl -u resin-supervisor --no-pager" logs using socket "unix\#/tmp/console.sock" "without" buffer
+...               AND             Get "journalctl -n 250 --no-pager" logs using socket "unix\#/tmp/console.sock" "with" buffer
```

Comment:
250 sounds enough, but might be worth just grabbing the lot unless it's going to be way too much to process, just in case!

Reply:
Solved this issue. Re-pushed the PR and now I am getting all the logs.